skimage/morphology/watershed.py

```diff
@@ -156,6 +156,9 @@ def watershed(image, markers, connectivity=1, offset=None, mask=None,
     compactness : float, optional
         Use compact watershed [3]_ with given compactness parameter.
         Higher values result in more regularly-shaped watershed basins.
+    watershed_line: bool, optional (default=False)
```

Comment:
Only in the signature. And, sometimes, in the description if it's not obvious what it means.

P.S. There is a discussion on changing this convention (https://github.com/scikit-image/scikit-image/issues/2409), but without any agreement atm.